test/e2e/fixtures/releases/helm-repository-no-rollback.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: podinfo-helm-repository
+  namespace: demo
+spec:
+  releaseName: podinfo-helm-repository
+  timeout: 30
+  wait: true
+  chart:
+    repository: https://stefanprodan.github.io/podinfo
+    name: podinfo
+    version: 3.2.0
```

Comment:
Should we use `4.0.1` here?

Reply:
Wouldn't hurt to update things to a newer podinfo version, but can be taken care of in a a separate PR to change the other fixtures as well.